### ISSUE TITLE
[1.12.2] Cable texture = facades

### ISSUE THREAD

**Issue by TheSnowyChickens** [AppliedEnergistics/Applied-Energistics-2#3442]
Hello,
at first: ITS NOT AN OPTIFINE BUG, it happend with and without.

The bug is very symple, maybe its just an option, so may convert this report to a Question. :D
The cable texture flicker through the facades (like the pictures below show)

#### Description
![2018-03-27_17 54 54](https://user-images.githubusercontent.com/29357962/37979227-5d4437c8-31e8-11e8-8606-ff5607b57429.png)

![2018-03-27_17 54 57](https://user-images.githubusercontent.com/29357962/37979228-5d601a88-31e8-11e8-8193-b381f0baa5c7.png)

![2018-03-27_17 55 00](https://user-images.githubusercontent.com/29357962/37979229-5d7d2d94-31e8-11e8-8c45-653791d2b9bf.png)

![2018-03-27_17 55 02](https://user-images.githubusercontent.com/29357962/37979230-5d9ec184-31e8-11e8-892f-8ae8eb44b0e1.png)

- Minecraft Version: 1.12.2
- AE2 Version: Stable 11 rv5
- Forge Version: 1.12.2-forge1.12.2-14.23.2.2638

**Comment by GuntherDW** [comment]
I'm not able to reproduce this, what is under that facade?

**Comment by yueh** [comment]
It only works, when the cable with the facade is a straight smart or covered cable (Have not tested a dense one).

It the usual z-fighting, but currently unsure how to fix it. As the cutout is basically too small and shout not overlap with the cable itself. Probably something to fix/change with the cable rework PR for rv6.

**Comment by GuntherDW** [comment]
As I said before, I can't replicate that specific behaviour. Neither in an obf'ed environment and my dev environment.

If you keep having it without mods like BetterFPS, OptiFine and others installed (double check with F3!), could you tell us what GPU you have installed?
I have an nvidia card (GTX 670) and do not have this issue.

**Comment by TheSnowyChickens** [comment]
This is now ONLY AE2 (appliedenergistics2-rv5-stable-11) on MC 1.12.2 in a NEW World with Forge (1.12.2-forge1.12.2-14.23.2.2654).
The last Picture is with F3 Screen, who you can see my Hardware. (Win10, 2 Monitors (Connected with DVI))

![2018-04-05_02 48 09](https://user-images.githubusercontent.com/29357962/38341997-e1653bda-387b-11e8-82d2-563bd22f6416.png)

![2018-04-05_02 48 07](https://user-images.githubusercontent.com/29357962/38342000-e4910794-387b-11e8-93ac-a2dbfc3a0c53.png)

![2018-04-05_02 48 03](https://user-images.githubusercontent.com/29357962/38342003-e619ef22-387b-11e8-80d0-fc2c17ec381d.png)

![2018-04-05_02 48 02](https://user-images.githubusercontent.com/29357962/38342006-e7959ba8-387b-11e8-8ffd-71b663128612.png)

![2018-04-05_02 48 00](https://user-images.githubusercontent.com/29357962/38342009-e88e7bec-387b-11e8-93f1-49607c231f8e.png)

![2018-04-05_02 51 48](https://user-images.githubusercontent.com/29357962/38342074-54a64a6c-387c-11e8-85b1-71b99beca68f.png)

My Win10 is on German. (Treiber -> Driver :) )
![grafik](https://user-images.githubusercontent.com/29357962/38342126-9dea2770-387c-11e8-9b6a-f8582e42dc05.png)

**Comment by Xiaminou** [comment]
I'm getting this too.

**Comment by Xiaminou** [comment]
https://youtu.be/A2nn52VxMsE

**Comment by TheSnowyChickens** [comment]
Yes, this darker lines on the edge of facades are not very beautiful too, I had it yesterday with white concrete and also good to see this cable - facade - texture issue in a video.

**Comment by Dogsteeves** [comment]
I also have a problem with this block and Facade neither are chiselled that the normal looking Laboratory block right-hand side
![screenshot_40](https://user-images.githubusercontent.com/34611694/41870046-e0713380-7888-11e8-9544-96d736908f69.png)

**Comment by TheSnowyChickens** [comment]
Hello again,

I thought with your new facade system, this issue would be eliminated, BUT no.
Just as Info.
1.12.2 with Stable 2 - Rv6 and Forge 2770 in the Stoneblock modpack. (Optifine = NO)

![2018-10-20_04 50 11](https://user-images.githubusercontent.com/29357962/47250580-c3e04980-d423-11e8-9b46-bd316dcdb0ec.png)
